etc/helm/examples/int-test-values-with-det.yaml

```diff
@@ -63,7 +63,7 @@ determined:
   enterpriseEdition: true
   useNodePortForMaster: false
   useNodePortForDB: false
-  detVersion: 0.23.1
+  detVersion: latest
```

Comment:
this may eventually lead to breakage, since the helm chart will lag behind for while longer. would hate to block PRs because we haven't addressed it as fast as we hope.

Reply:
I switched it back to a pinned version. I don't love either solution here. Either we are out of date or we likely break on an unseen update. I guess the lesser of two evils is to just need to manually update this version from time to time.